packages/website/components/accountPlansModal/accountPlansModal.js

```diff
@@ -8,12 +8,14 @@ import Modal from '../../modules/zero/components/modal/modal';
 import CloseIcon from '../../assets/icons/close';
 import AddPaymentMethodForm from '../../components/account/addPaymentMethodForm/addPaymentMethodForm.js';
 import { API, getToken } from '../../lib/api';
+import constants from '../../lib/constants.js';
 
-export async function putUserPayment(pm_id, plan_id) {
+export async function putUserPayment(agreement, pm_id, plan_id) {
```

Comment:
I think it's a good idea to add a jsdoc w/ type for the `agreement` value. I've ended up fixing several frontend bugs in all this that would have been caught by the typechecker. wdyt?

Reply:
e.g. I find myself wondering if `agreement` is allowed to be undefined, null, some other value here?

Reply:
fixed in https://github.com/web3-storage/web3.storage/pull/2028/commits/eb506e338c403f970ce0bb6c19fcc35d92ea04ec

(I got rid of this `putUserPayment` function and replaced it with the very similar fn in the api module)